roles/common/tasks/netdata.yml

```diff
@@ -0,0 +1,65 @@
+---
+# install netdata on node
+- name: Download the installation script
+  get_url:
+    url: https://my-netdata.io/kickstart.sh
+    dest: ~/kickstart.sh
+    mode: +x
+
+- name: Install Netdata
+  command: ~/kickstart.sh --dont-wait
+
+- name: Cleanup installation script
+  file:
+    path: ~/kickstart.sh
+    state: absent
+
+- name: Set hostname variable
+  set_fact:
+    hostname: "{{ inventory_hostname | default('localhost') }}"
```

Comment:
somewhere in the ansible.facts object hostname is already available, can we use that instead?
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html